src/test/java/com/o19s/es/explore/ExplorerQueryTests.java

```diff
@@ -135,7 +136,7 @@ public void testUniqueTerms() throws Exception {
         // Verify score is 4 (4 unique terms)
```

Comment:
Nitpicky, maybe update this comment since the score is changed. Also tq5 is no longer used, I included it before to double-check duplicate terms weren't bumping the unique term count.